### PR TITLE
Fix #530, Consider an extension of loudness info()

### LINKED ISSUE #530: Consider an extension of loudness info()

### PR DESCRIPTION
**Pull request by sunghee-hwang**
***
<a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/pull/535.html" title="Last updated on Jun 27, 2023, 9:15 PM UTC (e0cca1e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/535/3318721...e0cca1e.html" title="Last updated on Jun 27, 2023, 9:15 PM UTC (e0cca1e)">Diff</a>

### CODE REVIEW

**Review by cconcolato — COMMENTED** [review]
An alternative instead of defining reserved bytes for each flag is to reserve a single block and later split it when more flags are defined.

The semantics should indicate that these bytes shall be ignored by parsers compliant to this version of the specification.

File: index.bs [hunk 1]
```diff
@@ -1323,9 +1328,9 @@ class loudness_info() {
```
Comment (cconcolato) [inline index.bs:1328]: The sentence above says:
> info_type is a bitmask that specifies the type of OPTIONAL loudness information provided. The bits are set as follows, where the first bit is the LSB:

I don't understand why it says 'OPTIONAL'. Can we remove this word?

File: index.bs [hunk 2]
```diff
@@ -1314,6 +1314,11 @@ class loudness_info() {
       signed int (16) anchored_loudness;
     }
   }
+  if (info_type & 4 or info_typ & 8 or info_typ & 16 or info_typ & 32 or info_typ & 64 or info_typ & 128)) {
```
Comment (felicialim) [inline index.bs:1317]:
```suggestion
  if (info_type & 0b11111100 > 0)) {
```

File: index.bs [hunk 3]
```diff
@@ -1314,6 +1314,11 @@ class loudness_info() {
       signed int (16) anchored_loudness;
     }
   }
+  if (info_type & 0b11111100 > 0) {
```
Comment (tdaede) [inline index.bs:1317]: editorial nit: this is equivalent to info_type > 3

File: index.bs (hunk 1 above)
Comment (tdaede) [reply index.bs:1328]: We can maybe say "additional loudness information". I think it's to distinguish from integrated_loudness and digital_peak, as those are always provided.

**Review by cconcolato — APPROVED** [review]

**Pull request merged**